Formula/format-udf.rb

```diff
@@ -0,0 +1,14 @@
+class FormatUdf < Formula
+  desc "Bash script to format a block device to UDF"
+  homepage "https://github.com/JElchison/format-udf"
+  url "https://github.com/JElchison/format-udf/archive/1.4.0.tar.gz"
+  sha256 "41f259471fce19b8b716ff73498cd9e6864ffebc3855ffb5be5d763f5397d765"
+
+  def install
+    bin.install "format-udf.sh"
+  end
+
+  test do
+    system "#{bin}/format-udf.sh", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!